packages/gestalt/src/Video.jsdom.test.js

```diff
@@ -0,0 +1,186 @@
+// @flow
+import React from 'react';
+import { render } from '@testing-library/react';
+import Video from './Video.js';
+
+const A11Y_LABELS = Object.freeze({
+  accessibilityMaximizeLabel: 'Maximize',
+  accessibilityMinimizeLabel: 'Minimize',
+  accessibilityMuteLabel: 'Mute',
+  accessibilityPauseLabel: 'Pause',
+  accessibilityPlayLabel: 'Play',
+  accessibilityUnmuteLabel: 'Unmute',
+});
+
+describe('Video loading', () => {
+  beforeEach(() => {
+    window.HTMLMediaElement.prototype.load = () => {};
+  });
+
+  it('Does not load when string src does not change', () => {
+    const props = {
+      ...A11Y_LABELS,
+      aspectRatio: 1,
+      captions: 'https://media.w3.org/2010/05/sintel/captions.vtt',
+      src: 'https://media.w3.org/2010/05/sintel/trailer_hd.mp4',
+    };
+
+    const { container, rerender } = render(<Video {...props} />);
+    const spy = jest.spyOn(container.querySelector('video'), 'load');
```

Comment:
Do we automatically clear spies on teardown?

Reply:
The spy is scoped to the actual `test` so we it should not leak across others.